argocd/datadog_checks/argocd/check.py

```diff
@@ -0,0 +1,105 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from six import PY2
+
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheckV2
+from datadog_checks.base.constants import ServiceCheck
+
+from .config_models import ConfigMixin
+from .metrics import API_SERVER_METRICS, APPLICATION_CONTROLLER_METRICS, REPO_SERVER_METRICS
```

Comment:
Is there a convention to put metrics into a separate module? If no, what speaks against defining all those constants in this module?

Reply:
I think we normally do this. I'm not sure of the origin. But most if not all OpenMetrics checks have this format. I personally think it's easier to look at. Other examples:

https://github.com/DataDog/integrations-core/blob/master/istio/datadog_checks/istio/metrics.py